scripts/us_epa/air_emissions_inventory/common/us_air_pollution_emission_trends.py

```diff
@@ -0,0 +1,183 @@
+}
+
+
+class USAirPollutionEmissionTrends:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+
+    _mcf_template = ''
+    _tmcf_template = ''
+
+    def __init__(self,
+                 input_files: list,
+                 csv_file_path: str = None,
+                 mcf_file_path: str = None,
+                 tmcf_file_path: str = None) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._final_df = pd.DataFrame(columns=[
+            'geo_Id', 'year', 'SV_TEMP', 'observation', 'Measurement_Method'
+        ])
+
+    def set_cleansed_csv_file_path(self, cleansed_csv_file_path: str) -> None:
+        self._cleaned_csv_file_path = cleansed_csv_file_path
+
+    def set_mcf_file_path(self, mcf_file_path: str) -> None:
+        self._mcf_file_path = mcf_file_path
+
+    def set_tmcf_file_path(self, tmcf_file_path: str) -> None:
+        self._tmcf_file_path = tmcf_file_path
+
+    def aggregate_columns(self, df: pd.DataFrame, filter_values: list,
+                          measurement_method: str) -> pd.DataFrame:
+        """
+        Aggregates the columns based on SV
+
+        Args: 
+            df (pd.DataFrame): df as the input, to aggregate values
+
+        Returns:
+            df (pd.DataFrame): modified df as output
+        """
+        for i in filter_values:
+            df = df.drop(df[(df['SV_TEMP'].str.contains(i))].index)
+
+        # Replacing the columns for grouping as per
+        # StationaryFuelCombustion -    FuelCombustionElectricUtility
+        #                               FuelCombustionIndustrial
+        #                               EPA_FuelCombustionOther
+        # IndustrialAndOtherProcesses - ChemicalAndAlliedProductManufacturing
+        #                               MetalsProcessing
+        #                               PetroleumAndRelatedIndustries
+        #                               EPA_OtherIndustrialProcesses
+        #                               SolventUtilization
+        #                               StorageAndTransport
+        #                               WasteDisposalAndRecycling
+        # Transportation -              OnRoadVehicles
+        #                               NonRoadEnginesAndVehicles
+
+        df.loc[:, ('SV_TEMP')] = df['SV_TEMP'].replace(_SOURCE_GROUPS,
+                                                       regex=True)
+        df = df.groupby(['year', 'geo_Id', 'SV_TEMP']).sum().reset_index()
+        df['Measurement_Method'] = measurement_method
+        return df
+
+    def data_standardize(self, df: pd.DataFrame, column_name: str,
+                         standardize_values: dict) -> pd.DataFrame:
+        """
+        Replaces values of a single column into true values
+        from metadata returns the DF.
```

Comment:
nit: remove "returns the DF" or add a period before "returns the DF"

Reply:
removed returns the DF